sdks/python/apache_beam/transforms/core.py

```diff
@@ -1172,7 +1172,7 @@ class Windowing(object):
 
   def __init__(self, windowfn, triggerfn=None, accumulation_mode=None,
                output_time_fn=None):
-    global AccumulationMode, DefaultTrigger
```

Comment:
Let's remove the global out of this change until we know what it is supposed to do.